### DISCUSSION TITLE
Problems with segmentation with pre-trained models

### DISCUSSION THREAD

**Discussion by antoniaaaaaaaa** [Project-MONAI/MONAILabel#1511] [Q&A]
Failed to run inference in MONAI Label Server.
Message:: Status: 500; Response: Internal Server Error
![image](https://github.com/Project-MONAI/MONAILabel/assets/93753261/a5f6a829-ba06-4c20-a40d-0b200df22c2d)

**Comment by diazandr3s** [comment]
Hi @antoniaaaaaaaa,

It is difficult to know what was the issue from the slicer logs. Would you please paste the MONAI Label server logs to further investigate?
